app/model/DTO/Event/StatisticsItem.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Model\DTO\Event;
+
+final class StatisticsItem
```

Comment:
Potřebujeme tenhle objekt. Není to jen fancy náhrada za `array<string, int>`? Nebo počítáme s tím, že to poroste?

Reply:
může to růst, za mě je to pak výrazně čitelnější, když to není jen pole a vím k čemu to slouží. je to DTO s jasným pojmenováním

Reply:
Ok :+1: